### DISCUSSION TITLE
Temperature correction for myStrom

### DISCUSSION THREAD

**Discussion by basler-bebbi** [johannrichard/homebridge-dingz#482] [Q&A]
Hello. Great work!

In myStrom App a temperature correction can be defined.
Is there in the plug-in the same possibility too?

**Comment by johannrichard** [comment]
Hi, thanks for the comment and suggestion. This is not yet possible and has not been on my radar.

**Reply by johannrichard** [reply]
I'm tracking this in #519 -- however, beware, this might require quite a few changes and would require you to manually add each device to the plugin.
